breathecode/jobs/admin.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from django.contrib import admin
+from django.contrib import messages
+from .models import Platform, ZyteProject, Spider, Job, Employer, Position, PositionAlias, Tag, Location, LocationAlias
+from .actions import fetch_sync_all_data, run_spider, parse_date
+
+logger = logging.getLogger(__name__)
+
+
+# Register your models here.
+@admin.register(Platform)
+class PlatformAdmin(admin.ModelAdmin):
+    list_display = ('name', 'created_at')
+
+
+def fetch_sync_all_data_admin(modeladmin, request, queryset):
+    from django.contrib import messages
+    from .actions import fetch_sync_all_data
+    spiders = queryset.all()
+    try:
+        for s in spiders:
+            fetch_sync_all_data(s)
+            messages.success(request, f'{s.sync_desc}')
+    except Exception as e:
+        logger.debug(f'There was an error retriving the spider {str(e)}')
+        messages.error(request, f'There was an error retriving the spider {str(e)}')
+
+
+def run_spider_admin(modeladmin, request, queryset):
+    from django.contrib import messages
+    from .actions import run_spider
+    spiders = queryset.all()
+    try:
+        for s in spiders:
+            run_spider(s)
+            messages.success(request, f'The execution of the spider {s} was successful')
+    except Exception as e:
+        logger.debug(f'There was an error retriving the jobs {str(e)}')
+        messages.error(request, f'There was an error retriving the jobs {str(e)}')
+
+
+def parse_date_admin(modeladmin, request, queryset):
+    jobs = queryset.all()
+    from django.contrib import messages
```

Comment:
Up the imports to the first line of the function, just can be in the top of a import where inside a function the dockstrings